pkgs/development/cuda-modules/cuda/overrides.nix

```diff
@@ -220,27 +220,29 @@ attrsets.filterAttrs (attr: _: (builtins.hasAttr attr prev)) {
         else
           lib.getLib qt.qtwayland;
       qtWaylandPlugins = "${qtwayland}/${qt.qtbase.qtPluginPrefix}";
+      qt6Packages = final.pkgs.qt6Packages;
     in
     {
       # An ad hoc replacement for
       # https://github.com/ConnorBaker/cuda-redist-find-features/issues/11
       env.rmPatterns = toString [
-        "nsight-systems/*/*/libQt*"
+        "nsight-systems/*/*/libQt6*"
         "nsight-systems/*/*/libstdc*"
         "nsight-systems/*/*/libboost*"
         "nsight-systems/*/*/lib{ssl,ssh,crypto}*"
         "nsight-systems/*/*/lib{arrow,jpeg}*"
         "nsight-systems/*/*/Mesa"
         "nsight-systems/*/*/python/bin/python"
         "nsight-systems/*/*/libexec"
-        "nsight-systems/*/*/Plugins"
```

Comment:
Same question. These are prebuilt binaries which means they are presumed broken, so we do want to replace them with the respective ones from Nixpkgs.

Reply:
Agreed, but the additional code I added overwrites the Qt ones with links. These are the ones which remain:

```
/nix/store/k3vnvanjxvfr0jvkf0y6ijjka7nk2km3-nsight_systems-2023.2.3.1004-bin/nsight-systems/2023.2.3/host-linux-x64/Plugins/CorePlugin/libCorePlugin.so
/nix/store/k3vnvanjxvfr0jvkf0y6ijjka7nk2km3-nsight_systems-2023.2.3.1004-bin/nsight-systems/2023.2.3/host-linux-x64/Plugins/QuadDPlugin/libQuadDPlugin.so
/nix/store/k3vnvanjxvfr0jvkf0y6ijjka7nk2km3-nsight_systems-2023.2.3.1004-bin/nsight-systems/2023.2.3/host-linux-x64/Plugins/wayland-decoration-client/libbradient.so
/nix/store/k3vnvanjxvfr0jvkf0y6ijjka7nk2km3-nsight_systems-2023.2.3.1004-bin/nsight-systems/2023.2.3/host-linux-x64/Plugins/wayland-graphics-integration-client/libshm-emulation-server.so
/nix/store/k3vnvanjxvfr0jvkf0y6ijjka7nk2km3-nsight_systems-2023.2.3.1004-bin/nsight-systems/2023.2.3/host-linux-x64/Plugins/wayland-graphics-integration-client/libvulkan-server.so
/nix/store/k3vnvanjxvfr0jvkf0y6ijjka7nk2km3-nsight_systems-2023.2.3.1004-bin/nsight-systems/2023.2.3/host-linux-x64/Plugins/wayland-shell-integration/libwl-shell-plugin.so
/nix/store/k3vnvanjxvfr0jvkf0y6ijjka7nk2km3-nsight_systems-2023.2.3.1004-bin/nsight-systems/2023.2.3/host-linux-x64/Plugins/wayland-shell-integration/libxdg-shell.so
/nix/store/k3vnvanjxvfr0jvkf0y6ijjka7nk2km3-nsight_systems-2023.2.3.1004-bin/nsight-systems/2023.2.3/host-linux-x64/Plugins/wayland-shell-integration/libivi-shell.so
/nix/store/k3vnvanjxvfr0jvkf0y6ijjka7nk2km3-nsight_systems-2023.2.3.1004-bin/nsight-systems/2023.2.3/host-linux-x64/Plugins/wayland-shell-integration/libfullscreen-shell-v1.so
```